north-star-vision.md

```diff
@@ -0,0 +1,137 @@
+
+1. **Formalize supported release platforms (Introspectable)**
+
+   https://github.com/kubernetes/sig-release/issues/1337
+
+   Outcome: Definition of the life cycle for currently supported Kubernetes
+   artifacts and a guideline for the community about how to add new platforms.
+
+1. **Implement a Bill of Materials (BOM) for release artifacts (Introspectable /
+   Secure)**
+
+   https://github.com/kubernetes/release/issues/1837
+
+   Outcome: An automated formal verification of produced release artifacts for
+   every future release.
+
+1. **Enhance Kubernetes binary artifact management (Consumable)**
+
+   Outcome: Being able to promote files as artifacts and using this mechanism
+   for Kubernetes releases.
+
+1. **Define and collect metrics about Kubernetes releases (Introspectable)**
+
+   https://github.com/kubernetes/sig-release/issues/1527
+
+   Outcome: Being able to measure and interpret a set of defined metrics about
+   Kubernetes releases to associate actions with those.
+
+1. **Define and implement the release cadence survey (Introspectable)**
+
+   https://github.com/kubernetes/sig-release/issues/1526
+
+   Outcome: A regular survey evaluating the user experience of the current
+   release cadence.
+
+1. **Simplify CVE process for release management (Secure)**
+
+   https://github.com/kubernetes/sig-release/issues/896
+
+   https://github.com/kubernetes/release/issues/1354
+
+   Outcome: A documented and simple process for handling CVE information within
+   Kubernetes releases.
+
+1. **Establish Cluster API as first-class signal for upstream releases
+   (Secure)**
+
+   Outcome: Cluster API provides a CI signal for blocking release test jobs.
+
+1. **Enhance and simplify Kubernetes version markers (Consumable)**
+
+   Outcome: Clear documentation about available version markers as well as their
+   simplified automation.
+
+### Out of Scope
+
+Topic size estimates are roughly: small (1 week or less), medium (2-3 weeks),
+large (4-5 weeks).
+
+1. **Moving deb/rpm package builds to community infrastructure (large)**
+   - _What:_ The reb/rpm packages are built by the Google build admin, we should
+     move that to community infrastructure and build them automatically.
+   - _Why:_ It’s easier for us to fix issues independently that way and we have
+     more control about how the packages are being built.
+1. **Create a releases landing page (medium)**
+   - _What:_ Creating a releases website as a centralized and canonical place
+     for release related information.
+   - _Why:_ Finding consolidation options where information is right now spread
+     across different places. Helping people to find the latest patch release
+     and where to obtain its artifacts.
+1. **Signing of release artifacts (large)**
+   - _What:_ Being able to GPG sign release artifacts, which also includes
+     container images.
+   - _Why:_ This allows us to enhance the chain of trust between users of
+     Kubernetes artifacts.
+
```

Comment:
I have the feeling that all the out-of-scope topics should be in scope. They point deep and directly to the focus of north star. I would rather see for example the metrics topic more out of scope or nice to have thing, same like the release cadence survey.

Reply:
I moved every point in scope for now as multiple people mentioned.